ThirdPartyNotices.txt

```diff
@@ -2,7 +2,7 @@
 THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
-Work Item Details Widget Extension incorporates third party material from the projects listed below. The original copyright notice and the license under which Microsoft received such third party material are set forth below. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.  
+Work Item Details Widget Extension incorporates third party material from the projects listed below. The original copyright notice and the license under which ALM | DevOps Ranger Contributors received such third party material are set forth below. ALM | DevOps Ranger Contributors reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.  
```

Comment:
Please drop the s after reserves in:

ALM | DevOps Ranger Contributors reserve**s** ...

Reply:
got it. Note it was in the original thirdpartynotice file. Might be worthwhile to check the initial template too.